Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
I use `if` to be explicit, I think the ternary should only be used in expressions and not as a statement by itself. About setters and getters, was contemplating that but settled on this as it plays into c++ initializer form and looks better I think.